Source/Core/DolphinWX/ARCodeAddEdit.cpp

```diff
@@ -44,8 +44,8 @@ CARCodeAddEdit::CARCodeAddEdit(int _selection, const std::vector<ActionReplay::A
 	}
 	else
 	{
-		currentName = StrToWxStr(arCodes.at(selection).name);
-		tempEntries = arCodes.at(selection);
+		currentName = StrToWxStr(arCodes->at(selection).name);
+		tempEntries = arCodes->at(selection);
```

Comment:
It would need to be `(*arcodes)[selection]`.  Not sure if that's an improvement.